doc/templates/uno/component/navbar.scss

```diff
@@ -119,13 +115,28 @@
         font-size: 19px;
         line-height: 17px;
       }
+
```

Comment:
@marcoscodnet with the latest fixes, here are my current comments:
- The small "Sign In" button seems a bit odd with the bigger "Get Started" button.
   ![image](https://github.com/user-attachments/assets/1119f723-522c-4b39-8ab4-1cf6a81e1514)
   ![image](https://github.com/user-attachments/assets/89d07c0e-e1d9-4b9e-8e36-cccf5fc36a2d)
   
- For the more narrow/phone breakpoint can we at least have the same width/height for the "get started" and "sign in" button, it looks odd at the moment:
    ![image](https://github.com/user-attachments/assets/939bfde5-858e-4704-a315-e94a8cc5d551)